include/clap/ext/draft/triggers.h

```diff
@@ -0,0 +1,132 @@
+#pragma once
+
+#include "../../plugin.h"
+#include "../string-sizes.h"
+
+static CLAP_CONSTEXPR const char CLAP_EXT_TRIGGERS[] = "clap.triggers.draft/0";
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+/// @page Trigger events
+///
+/// This extension enables the plugin to provide a set of triggers to the host.
+///
+/// Some examples for triggers:
+/// - start recording into a plugin-internal audio/note buffer
+/// - trigger a retrospective audio/note looper
+/// - trigger a sample-and-hold unit (maybe even per-voice)
+/// - retrigger an envelope during voice lifetime
+/// - shake event for a spring-reverb

```

Comment:
This example is interesting, because when you shake a spring reverb you may want an intensity, etc...

So two options, maybe this example doesn't fit or the trigger event needs additional data?

I think that the trigger is fine without data, and for triggering the spring, you may want instead to use a note port or notes on a specific channel.

Reply:
Also already thought about this point. What's your personal opinion? In the end we could achieve the trigger functionality with notes, just ignoring note-offs and all the data we don't need. So is a triggers extension useful/special enough to justify its existence?

Reply:
I think it's better to just remove that example to avoid confusion. So I'll do that.

Reply:
Trigger is still cool, you can show a button for it in the host gui, ...